qcodes_contrib_drivers/drivers/CopperMountain/M5180.py

```diff
@@ -0,0 +1,572 @@
+# This Python file uses the following encoding: utf-8
+# Etienne Dumur <etienne.dumur@gmail.com>, august 2020
+# Simon Zihlmannr <zihlmann.simon@gmail.com>, february/march 2021
+import logging
+import numpy as np
+from typing import Tuple, Optional
+
+from qcodes import VisaInstrument
+from qcodes.instrument.base import InstrumentBase
+from qcodes.utils.validators import Numbers, Enum, Ints
+from qcodes.utils.helpers import create_on_off_val_mapping
+
+from qcodes.instrument.parameter import (
+    MultiParameter,
+    ParamRawDataType
+)
+
+log = logging.getLogger(__name__)
+
+class FrequencySweepMagPhase(MultiParameter):
+    """
+    Sweep that returns magnitude and phase.
+    """
+
+    def __init__(self,
+        name: str,
+        start: float,
+        stop: float,
+        npts: int,
+        instrument: "M5180",
+        ) -> None:
+        """
+        Linear frequency sweep that returns magnitude and phase for a single
+        trace.
+
+        Args:
+            name (str): Name of the linear frequency sweep
+            start (float): Start frequency of linear sweep
+            stop (float): Stop frequency of linear sweep
+            npts (float): Number of points of linear sweep
```

Comment:
npts is an int. Since the types are given in the signature you are also free to completely remove them from the docstring

Reply:
true, changed to int